style.css

```diff
@@ -386,6 +386,144 @@
     display: grid;
     grid-template-columns: 1fr;
   }
+
+  #works_popup_mobile {
+    display: none;
+  }
+
+  #works_popup_desktop {
+    visibility: hidden;
+    position: fixed;
+    width: 100%;
+    height: 100vh;
+    top: 0;
+    background-color: rgba(0, 0, 0, 0.4);
+    padding: 0;
+    overflow-y: scroll;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .project1_popup {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    background-color: #fff;
+    border-radius: 16px;
+    margin: 20px;
+    opacity: none;
+    margin-left: 10%;
+    margin-right: 15%;
+  }
+
+  .popup_header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    align-items: center;
+    margin-top: 0;
+  }
+
+  .popup_img_desktop {
+    display: flex;
+    align-items: center;
+    width: 100%;
+  }
+
+  .popup_description_desktop_container {
+    width: 80%;
+    display: flex;
+  }
+
+  .project1_popup_container {
+    margin-left: 10px;
+    margin-right: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .project1_title_popup_dekstop {
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-bottom: 0;
+    color: #172b4d;
+    flex-direction: row;
+  }
+
+  .desktop_popup_positioner {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #close_project_popup_desktop {
+    font-size: 25px;
+    border: none;
+    font-weight: bold;
+    background-color: #fff;
+    margin-top: 0;
+  }
+
+  .small_button_project1_popup {
+    margin-right: 35%;
+  }
+
+  .project1_popup_buttons {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    margin-bottom: 20px;
+  }
+
+  .mobile_popup_seperator {
+    margin-bottom: 20px;
+  }
+
```

Comment:
- [ ] Kindly make sure your Popup Window design is close as possible to the one from the Figma template, **both on Desktop and Mobile,** as required for the project(e.g.: background color, blur effect on the background, text color, spacing around the popup window(top && bottom), the close Popup button,paddings,...) Make sure you check every detail from Figma to fix this styling issue.
### Figma Design
![Screenshot from 2022-07-20 12-25-23](https://user-images.githubusercontent.com/83042742/179961284-3e453f38-979b-40c5-b106-1d1a27b67ad6.png)
### Your Desig
![Screenshot from 2022-07-20 12-35-22](https://user-images.githubusercontent.com/83042742/179961966-5ff75e36-7736-4ba5-a7a6-b11c12c9bd5b.png)
n